src/lib/molecules/Charts/MapChart/index.js

```diff
@@ -71,7 +71,8 @@ function MapChart({
             chart: {
                 width,
                 height,
-                type: 'map'
+                type: 'map',
+                animation: false
```

Comment:
The change will affect all animations in the Map Chart component, but the issue is only for blinking animation when the data prop is changed. e.g. this change will remove zoom in/out animation too.